lib/network.sh

```diff
@@ -12,8 +12,15 @@ function network_address() {
   resultvar=$1
   network=$2
   record=$3
+  OS=$(awk -F= '/^ID=/ { print $2 }' /etc/os-release | tr -d '"')
+
+  if [ "${OS}" == "ubuntu" ]
+  then
+    result=$(python3 -c "import ipaddress; import itertools; print(next(itertools.islice(ipaddress.ip_network(u\"$network\").hosts(), $record - 1, None)))")
+  else
+    result=$(python -c "import ipaddress; import itertools; print(next(itertools.islice(ipaddress.ip_network(u\"$network\").hosts(), $record - 1, None)))")
```

Comment:
Did you look at the alternatives approach I mentioned to avoid this conditional?

Reply:
yes, done